packages/storybook/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "storybook",
+  "version": "0.0.0",
+  "license": "Apache-2.0",
+  "private": true,
+  "backstage": {
+    "role": "frontend"
+  },
+  "sideEffects": false,
+  "scripts": {
+    "start": "storybook dev -p 6006",
+    "start:debug": "yarn start:storybook --debug-webpack",
+    "build": "storybook build",
+    "lint": "backstage-cli package lint",
+    "clean": "backstage-cli package clean"
+  },
+  "devDependencies": {
+    "@backstage/cli": "0.23.0",
+    "@storybook/addon-controls": "^7.5.3",
+    "@storybook/addon-essentials": "^7.5.3",
+    "@storybook/addon-interactions": "^7.5.3",
+    "@storybook/addon-links": "^7.5.3",
+    "@storybook/addon-onboarding": "^1.0.8",
+    "@storybook/blocks": "^7.5.3",
+    "@storybook/react": "^7.5.3",
+    "@storybook/react-webpack5": "^7.5.3",
+    "@storybook/testing-library": "^0.2.2",
+    "@testing-library/jest-dom": "^5.10.1",
```

Comment:
```suggestion
    "@testing-library/jest-dom": "^5.17.0",
```
Other packages are using this version. Can we do the same? It would be easier to bump versions at once.

Reply:
IDK. I'll owe you an answer on this one, I'll check it and I'll get back to you.

Reply:
![image](https://github.com/caponetto/backstage-plugins/assets/77008341/0e1b524c-7659-4dd6-86a5-d8672312b11f)

Long story short, what will get installed eventually is `5.17.0` (by the time of this message or until they release a new version).

Reply:
Requesting `^5.10.1` means that at least this package needs a version of `@testing-library/jest-dom` that is no older than 5.10.1.

Reply:
addressed in https://github.com/caponetto/backstage-plugins/pull/24/commits/f064fe4b27689f062d8d63ce370873ace0c26c6b